html/elements/command.json

```diff
@@ -0,0 +1,359 @@
+{
+  "html": {
+    "elements": {
+      "command": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/command",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Before Firefox 24, although not implemented, an object of class &lt;HTMLCommandElement&gt; was created, instead of the compliant &lt;HTMLUnknownElement&gt;."
```

Comment:
I think you meant to put this as `<code>HTMLCommandElement</code>` etc. (same below for firefox_android)

Reply:
Fixed.

Reply:
Not quite, the `&lt;` and `&gt;` are not needed. This is an interface, not an element.

Reply:
Fixed, this time